src/App/Controls/PasswordStrengthProgressBar/PasswordStrengthProgressBar.xaml.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Runtime.CompilerServices;
+using Xamarin.Forms;
+
+namespace Bit.App.Controls
+{
+    public partial class PasswordStrengthProgressBar : StackLayout
+    {
+        private PasswordStrengthViewModel ViewModel => BindingContext as PasswordStrengthViewModel;
+
+        public static readonly BindableProperty VeryWeakColorProperty = BindableProperty.Create(
+            nameof(VeryWeakColor),
+            typeof(Color),
+            typeof(PasswordStrengthProgressBar),
+            Utilities.ThemeManager.GetResourceColor("DangerColor"));
+
+        public static readonly BindableProperty WeakColorProperty = BindableProperty.Create(
+            nameof(WeakColor),
+            typeof(Color),
+            typeof(PasswordStrengthProgressBar),
+            Utilities.ThemeManager.GetResourceColor("WarningColor"));
+
+        public static readonly BindableProperty GoodColorProperty = BindableProperty.Create(
+            nameof(GoodColor),
+            typeof(Color),
+            typeof(PasswordStrengthProgressBar),
+            Utilities.ThemeManager.GetResourceColor("PrimaryColor"));
+
+        public static readonly BindableProperty StrongColorProperty = BindableProperty.Create(
+            nameof(StrongColor),
+            typeof(Color),
+            typeof(PasswordStrengthProgressBar),
+            Utilities.ThemeManager.GetResourceColor("SuccessColor"));
```

Comment:
Remove default color gotten from `ThemeManager`, it's an extra overhead that won't be used given the global style